exercises/practice/pov/test/pov_test.exs

```diff
@@ -0,0 +1,148 @@
+defmodule PovTest do
+  use ExUnit.Case
+
+  def equal_trees({a, children_a}, {a, children_b}) do
+    if length(children_a) == length(children_b) do
+      sorted_a = Enum.sort(children_a)
+      sorted_b = Enum.sort(children_b)
+
+      Enum.zip_with(sorted_a, sorted_b, &equal_trees/2)
```

Comment:
The CI is already reporting that tests don't pass in Elixir 1.8 😢. It's because this function was introduced in Elixir 1.12. To allow as many students as possible to solve the exercises, we have a CI check that runs the example solutions against different versions of Elixir, starting at 1.7. I think it won't be a problem to rewrite this with `zip` and `map` instead, right?

Reply:
No problem indeed.